tree.go

```diff
@@ -282,6 +283,53 @@ func (t *Tree) insert(
 	)
 }
 
+// InsertRange is the same as Insert, except it will insert all subnets within
+// the range of IPs specified by `[start,end]`.
+func (t *Tree) InsertRange(
+	start net.IP,
+	end net.IP,
+	value mmdbtype.DataType,
+) error {
+	return t.InsertRangeFunc(start, end, t.inserterFuncGen(value))
+}
+
+// InsertRangeFunc is the same as InsertFunc, except it will insert all subnets
+// within the range of IPs specified by `[start,end]`.
+func (t *Tree) InsertRangeFunc(
+	start net.IP,
+	end net.IP,
+	inserter inserter.InserterFunc,
+) error {
+	return t.insertRange(start, end, recordTypeData, inserter, nil)
+}
+
+func (t *Tree) insertRange(
+	start net.IP,
+	end net.IP,
+	recordType recordType,
+	inserter inserter.InserterFunc,
+	node *node,
+) error {
+	_start, ok := netaddr.FromStdIP(start)
+	if !ok {
+		return errors.New("start IP is invalid")
+	}
+	_end, ok := netaddr.FromStdIP(end)
+	if !ok {
+		return errors.New("end IP is invalid")
+	}
+
+	r := netaddr.IPRangeFrom(_start, _end)
```

Comment:
Maybe it would be worth returning an error if `r.IsValid()` is false.

Reply:
Added in https://github.com/maxmind/mmdbwriter/pull/25/commits/b00a7125e80388b28ca99414106e10a72d2df01c